hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ResourceAttributeEnricher.java

```diff
@@ -63,6 +71,18 @@ public void enrichEvent(StructuredTrace trace, Event event) {
                       }
```

Comment:
Deployment-type condition should be added after this condition IMO.

Reply:
It has to be a separate if block. Not sure how we can embed it in above statemnet